### ISSUE TITLE
False identification

### ISSUE THREAD

**Issue by victornoel** [dependency-check/DependencyCheck#365]
Hi,

I get a false positive for an identification and I don't get why, there is nothing that seems to obvious to me.
The artefact is
http://search.maven.org/#search|ga|1|1%3A%22f862e6a191a256861863d8c47084ca9bd93f486d%22

And it is identified as 

```
<identifier type="cpe" confidence="HIGHEST">
    <name>(cpe:/a:apache:camel:1.0.0)</name>
    <url>https://web.nvd.nist.gov/view/vuln/search-results?adv_search=true&amp;cves=on&amp;cpe_version=cpe%3A%2Fa%3Aapache%3Acamel%3A1.0.0</url>
</identifier>
```

I'm using the Jenkins plugin, version 1.3.1.

**Comment by jeremylong** [comment]
dependency-check will ocassionally report false positives - this is due to [how it works](http://jeremylong.github.io/DependencyCheck/general/internals.html). Because dependency-check use textual data (refered to as evidence) from the dependencies being analyzed for library identification the match is sometimes wrong. This is generally an easy onboarding task when you first start scanning a project - if any of the matches seem completely off due two things:
1. Report the false positive as an issue (Thanks!)
2. Until the issue is resolve, you can use a local [suppression file](http://jeremylong.github.io/DependencyCheck/general/suppression.html) to remove these findings from the report.

I have added a [suppression rule](https://github.com/jeremylong/DependencyCheck/commit/ba6d3bbe152748a95935cf562b96653b2115a30d) to the core engine for this false positive and it will be included in the next release.

Best Regards,

Jeremy Long
